src/index.tsx

```diff
@@ -110,6 +110,19 @@ export const Player = forwardRef<PlayerRef, Props>(function Player(
         }
     }, [])
 
+    useEffect(() => {
+        if (frame.current && replayer.current) {
+            // Only add the events that don't already exist in replayer's context
+            const numCurrentEvents =
+                replayer.current?.service.state.context.events.length ?? 0
+            const eventsToAdd = props.events.slice(numCurrentEvents) ?? []
+            eventsToAdd.forEach((event) => replayer.current?.addEvent(event))
+
+            const meta = replayer.current.getMetaData()
+            setMeta(meta)
```

Comment:
I think we should add a `duration` and `isBuffering` prop, and we display the passed in duration until the full recording has buffered. Otherwise, I think it will be an odd experience where the session duration keeps updating (and the seek bar jumps) each time more data is loaded.